napari/layers/_base_layer/model.py

```diff
@@ -119,14 +121,18 @@ def coordinates(self):
         The setter expects the a 2-tuple of coordinates in canvas space
         ordered (x, y) and then transforms them to image space and inserts
         them into the correct position of the layer indices. The length of the
-        tuple is equal to the number of dimensions of the layer.
+        tuple is equal to the number of dimensions of the layer. If `None` is
+        passed then (0, 0) in will replace necessary the layer indices.
```

Comment:
the "then (0, 0) in" confuses me, is this a typo or am i just bad at english?

Reply:
type - "in" should not be there!

Reply:
*typo